src/main/java/net/glowstone/entity/GlowPlayer.java

```diff
@@ -818,8 +818,35 @@ public void pulse() {
             }
         }
 
-        if (foodLevel == 0 && getHealth() > 1 && ticksLived % 80 == 0) {
-            damage(1, DamageCause.STARVATION);
+        // Process food level and starvation based on difficulty.
+        switch (world.getDifficulty()) {
+            case PEACEFUL: {
+                if (foodLevel < 20 && ticksLived % 20 == 0) {
+                    foodLevel++;
+                }
+                break;
+            }
+            case EASY: {
+                if (foodLevel == 0 && getHealth() > 10 && ticksLived % 80 == 0) {
+                    damage(1, DamageCause.STARVATION);
+                }
+                break;
+            }
+            case NORMAL: {
+                if (foodLevel == 0 && getHealth() > 1 && ticksLived % 80 == 0) {
+                    damage(1, DamageCause.STARVATION);
+                }
+                break;
+            }
+            case HARD: {
+                if (foodLevel == 0 && ticksLived % 80 == 0) {
+                    damage(1, DamageCause.STARVATION);
+                }
+                break;
+            }
+            default: {
+                // Do nothing when there're other game difficulties.
```

Comment:
I would prefer if you didn't use a contraction here.

Reply:
You mean like this?
```java
        // Process food level and starvation based on difficulty.
        switch (world.getDifficulty()) {
        case PEACEFUL: {
            if (foodLevel < 20 && ticksLived % 20 == 0) {
                foodLevel++;
            }
            break;
        }
        case EASY: {
            if (foodLevel == 0 && getHealth() > 10 && ticksLived % 80 == 0) {
                damage(1, DamageCause.STARVATION);
            }
            break;
        }
        case NORMAL: {
            if (foodLevel == 0 && getHealth() > 1 && ticksLived % 80 == 0) {
                damage(1, DamageCause.STARVATION);
            }
            break;
        }
        case HARD: {
            if (foodLevel == 0 && ticksLived % 80 == 0) {
                damage(1, DamageCause.STARVATION);
            }
            break;
        }
        default: {
            // Do nothing when there're other game difficulties.
        }
        }
```

Reply:
No I meant just use `there are` instead of `there're`

Reply:
No I meant just use `there are` instead of `there're`